easy-install.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import os
+import sys
+import time
+from shutil import which
+import platform
+from secrets import token_bytes
+from base64 import b64encode
+
+CRED = "\033[31m"
+CEND = "\033[0m"
+CGRN = "\33[92m"
+CYLW = "\33[93m"
+
+
+def clone_frappe_docker_repo() -> None:
+    try:
+        subprocess.run(
+            ["git", "clone", "https://github.com/frappe/frappe_docker"], check=True
+        )
+    except Exception as e:
+        print(f"\n{CRED}Cloning frappe_docker Failed{CEND}\n\n", e)
+
+
+def write_to_env(wd, site,db_pass,email):
+    site_name = site if site != "" else ""
+    with open(os.path.join(wd, ".env"), "w") as f:
+        f.writelines(
+            [
+                "FRAPPE_VERSION=v14.17.1\n",
+                "ERPNEXT_VERSION=v14.9.0\n",
+                f"DB_PASSWORD={db_pass}\n",
+                "DB_HOST=db\n",
+                "DB_PORT=3306\n",
+                "REDIS_CACHE=redis-cache:6379\n",
+                "REDIS_QUEUE=redis-queue:6379\n",
+                "REDIS_SOCKETIO=redis-socketio:6379\n",
+                f"LETSENCRYPT_EMAIL={email}\n"
+                f"FRAPPE_SITE_NAME_HEADER={site_name}",
+            ]
+        )
+
+def generate_pass(length:int=9) -> str:
+    return b64encode(token_bytes(length)).decode()
```

Comment:
Note: `token_bytes` will give 2x length for specified length. If not intentional copy `generate_hash` from frappe to get exact length.